csrc/scheduler/utils.cpp

```diff
@@ -2005,8 +1991,7 @@ void propagateViewTransforms(Fusion* fusion, const ComputeAtMap& ca_map) {
 }
 
 bool isFastestDimReduction(TensorView* tv) {
-  for (auto it = tv->getMaybeRFactorDomain().rbegin();
-       it != tv->getMaybeRFactorDomain().rend();
+  for (auto it = tv->getRootDomain().rbegin(); it != tv->getRootDomain().rend();
```

Comment:
I think using root domain is more aligned with the deleted comment:
```
  // This is backwards from how we normally think about grabbing root dimensions
  // to process. If we're in a reduction scheduler and we're using the rfactored
  // reduction tensor view, we don't care about the rfactor domain, we care
  // about the root domain because we're looking to vectorize the reads (input
  // tensor views). Otherwise we do want the rfactor domain. So this is the
  // reverse of our typical check, we actually want to selectively ignore the
  // rfactor domain.
```

Reply:
Shouldn't this be the allocation domain because that defines which domains are fastest

Reply:
Yes, I think so, I will change this to allocation domain, although I haven't started supporting allocation domain for schedulers yet so simply changing this to allocation domain does not make things better in practice.

Reply:
Changed